### DISCUSSION TITLE
'<' not supported between instances of 'int' and 'NoneType'

### DISCUSSION THREAD

**Discussion by marp188** [Flexget/Flexget#2887] [Q&A]
Hello

After completely rebuilfing the flexget database from scratch, I started again received this arcane error message:

"Unhandeled error in plugin .... '<' not supported between instances of 'int' and 'NoneType' 

Any ideeas?

**Comment by paranoidi** [comment]
It reads in the meesage. It's a bug. Open a ticket.
